### PR TITLE
Added photo 

### PR DESCRIPTION
**Pull request by faizahpathan**
Added photo and social media info for FOSSASIAA GCI.

### CODE REVIEW

**Comment by faizahpathan** [comment]
Can someone tell me my error?

File: _data/students.json [hunk 1]
```diff
@@ -423,5 +423,12 @@
     },
     "imgName": "izzylys.jpg",
     "name": "Izzy Lyseggen"
-}   
+},
+{ 
+    "social": {
+        "github": "Faizahpathan"
+    }
```
Comment (roonyh) [inline _data/students.json]: you need a comma here. unless its not valid json.

**Comment by faizahpathan** [comment]
Is there anything else for me to do?

**Comment by roonyh** [comment]
A rebase (http://stackoverflow.com/questions/17182624/contributing-to-project-on-github-how-to-rebase-my-pull-request-on-top-of-mast) would be really helpful.

**Comment by mariobehling** [comment]
@RoonyH I think it would be best to get a rebase. We labeled this task as a beginner task. As the contest is ending today, let's accept it. It would be great, if you could help to merge it. Thank you.

**Comment by roonyh** [comment]
Sure. I will merge this.

**Pull request merged**